utils/agent_remote.go

```diff
@@ -346,7 +347,7 @@ func CheckAgentErrorsOnSegments(c *cluster.Cluster, fpInfo filepath.FilePathInfo
 	}
 	if numErrors > 0 {
 		helperLogName := fpInfo.GetHelperLogPath()
-		return errors.Errorf("Encountered errors with %d helper agent(s).  See %s for a complete list of segments with errors, and see %s on the corresponding hosts for detailed error messages.",
+		return errors.Wrapf(AgentErr, "Encountered errors with %d helper agent(s).  See %s for a complete list of segments with errors, and see %s on the corresponding hosts for detailed error messages.",
```

Comment:
Wrapf returns an error with a stack trace. Do we need  stack trace? Can we use another function, which doesn't collect  stack trace?

Reply:
> Wrapf returns an error with a stack trace.

like `Errorf` does

> Can we use another function, which doesn't collect stack trace?

`WithMessagef` annotates `err` with the `format` specifier



> Do we need stack trace?

Maybe we need in future